externs/browser/w3c_css.js

```diff
@@ -2097,24 +2097,34 @@ Window.prototype.scrollY;
 Window.prototype.pageYOffset;
 
 /**
- * @param {number} x
- * @param {number} y
+ * @typedef {{
+ *   left: (number|undefined),
+ *   top: (number|undefined),
+ *   behavior: (string|undefined)
+ * }}
+ * @see https://www.w3.org/TR/cssom-view/#dictdef-scrolltooptions
+ */
+var ScrollToOptions;
+
+/**
+ * @param {number|!ScrollToOptions} x
```

Comment:
Instead of x, I'll rename this to scrollToOptionsOrX, just to make it clear what it is. 

Similar to https://github.com/google/closure-compiler/blob/2375110b30ccb83b0a550c8c3d46c526e3896d62/externs/browser/w3c_css.js#L2322

No need to change it on your side, I'll just change it in the internal version.